docs/api/IoT.js-API-Assert.md

```diff
@@ -5,203 +5,230 @@ The following shows Assert module APIs available for each platform.
 |  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) |
 | :---: | :---: | :---: | :---: |
 | assert.assert | O | O | O |
-| assert.doesNotThrows | O | O | O |
+| assert.doesNotThrow | O | O | O |
 | assert.equal | O | O | O |
 | assert.fail | O | O | O |
 | assert.notEqual | O | O | O |
 | assert.notStrictEqual | O | O | O |
 | assert.strictEqual | O | O | O |
 | assert.throws | O | O | O |
 
+# Assert
 
-### Contents
+Assert module is designed for writing tests for applications.
 
-- [Assert](#assert)
-    - [Module Functions](#module-functions)
-        - [`assert.assert(value[, message])`](#assertassertvalue-message)
-        - [`assert.doesNotThrows(block[, message])`](#assertdoesnotthrowsblock-message)
-        - [`assert.equal(actual, expected[, message])`](#assertequalactual-expected-message)
-        - [`assert.fail(actual, expected, message, operator)`](#assertfailactual-expected-message-operator)
-        - [`assert.notEqual(actual, expected[, message])`](#assertnotequalactual-expected-message)
-        - [`assert.notStrictEqual(actual, expected[, message])`](#assertnotstrictequalactual-expected-message)
-        - [`assert.strictEqual(actual, expected[, message])`](#assertstrictequalactual-expected-message)
-        - [`assert.throws(block[, expected, message])`](#assertthrowsblock-expected-message)
-    - [Class: assert.AssertionError](#class-assertassertionerror)
-        - [Properties](#properties)
-            - [`actual`](#actual)
-            - [`message`](#message)
-            - [`expected`](#expected)
-            - [`name`](#name)
-            - [`operator`](#operator)
+You can access the functions of the module by adding `require('assert')` to your file.
 
 
-# Assert
+## Class: AssertionError
+
+Assert module will produce `AssertionError` in case of an assertion failure. `AssertionError` inherits standard `Error` thus it has properties provided by `Error` object including additional properties.
+
 
-Assert module is for writing test for applications.
+### Properties
```

Comment:
Please check #873 - 6. Remove all the section titles such as Class Methods, Prototype Function, Module Functions/Properties, Properties, Event, and so on.

Please refer to https://github.com/bzsolt/iotjs/blob/ac74208d8c2cd1453d8ec7bc561cf48c687137fc/docs/api/IoT.js-API-HTTP.md#messagestatuscode for Properites.